app/src/main/java/org/breezyweather/common/source/LocationPreset.kt

```diff
@@ -75,7 +75,7 @@ enum class LocationPreset(
     // Don’t add cwa for TAIWAN as it is a rate-limited source
     CHINA("china"),
     HONG_KONG("hko"),
-    INDONESIA("bmkg", pollen = "openmeteo", minutely = "openmeteo", normals = "accu"),
+    INDONESIA("bmkg", minutely = "openmeteo", normals = "accu"),
```

Comment:
Are you sure Open-Meteo doesn't work as a pollen source in Indonesia during the pollen season?

Reply:
Almost sure. For countries like Israel and Türkiye, the OpenMeteo API call returns `0` for pollen concentration now; where as for Indonesia, the API call returns `null`.

Given that OpenMeteo officially says that only Europe is supported for pollen data, I think it's sensible to remove it for Indonesia. We can monitor it from time to time, and add back if it starts unofficially supporting other countries.

In any case, the main concern with air in Indonesia is not so much pollen. It's volcanic ash. 🌋😷